.github/workflows/ci.yml

```diff
@@ -132,3 +132,50 @@ jobs:
     - name: Upload dev docs
       run: rsync -az --delete doc/build/html/ docs@${{ secrets.server_ip }}:/usr/share/nginx/pandas/pandas-docs/dev
       if: github.event_name == 'push'
+
+  build-sdist:
+    name: Build from source distribution
+    runs-on: ubuntu-latest
+    steps:
+    - name: Update for build environment
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y build-essential
+        sudo apt-get clean
+    - name: Setting conda path
+      run: echo "$CONDA/bin" >> $GITHUB_PATH
+    - name: Update conda
+      run: |
+        conda config --set quiet true --set always_yes true
+        conda update -n base -c defaults conda
+        conda list
+    - name: Checkout pandas
+      uses: actions/checkout@v2
```

Comment:
I'm not sure if it's related here. But using checkout@v2 caused a lot of failures and errors in #38344.

Reply:
maybe the cause of the version issues. I added the step to pull the tags but that didn't seem to fix.

I don't have the issue in #36704, but am adding tag explicitly there and using fetch-depth with the checkout@v2 action.